PyInstaller/hooks/hook-django.py

```diff
@@ -90,14 +112,16 @@
     installed_apps = eval(get_module_attribute(package_name + '.settings', 'INSTALLED_APPS'))
     migration_modules.extend(set(app + '.migrations' for app in installed_apps))
     # Copy migration files.
-    for mod in migration_modules:
-        mod_name, bundle_name = mod.split('.', 1)
-        mod_dir = os.path.dirname(get_module_file_attribute(mod_name))
-        bundle_dir = bundle_name.replace('.', os.sep)
-        pattern = os.path.join(mod_dir, bundle_dir, '*.py')
-        files = glob.glob(pattern)
-        for f in files:
-            datas.append((f, os.path.join(mod_name, bundle_dir)))
+    datas += collect_modules_files(migration_modules)
```

Comment:
As far as I can see, the removed lines removed are just moved into the function above. Why not just use `utils.hookcollect_submodules()` here and drop the whole function?

Anyway: Since this change is unrelated to Django 2.0, it should go into a separate commit.

Reply:
This code refactoring is necessary to be able to do the same for commands_modules in lines bellow.

I'll have a look to `utils.hookcollect_submodules()` if it can do the job